src/chapters/introduction-and-setup/naming-conventions.rst

```diff
@@ -28,6 +28,6 @@ rules <https://msdn.microsoft.com/en-us/library/aa664670(v=vs.71).aspx>`__.
 
 
 Microsoft provides some `more detailed naming
-conventions <https://msdn.microsoft.com/en-us/library/ms229002.aspx>`__.
+conventions <https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/identifier-names>`__.
```

Comment:
Hi Scott.  Could you provide some context behind your suggested change to this link?  It's only one level up from the original, but the link to the naming conventions page is easy to overlook.

Reply:
The updated link goes into much more detail, however looking at the dates I think the existing link is more up to date.

Reply:
The old link is out of date, from 2008.  The new link is the updated naming conventions from 2018.

Reply:
The 2018 page contains pretty much the same information given in the textbook, but it mentions things we're not ready for in chapter 1 (e.g. Enums).  Since it contains a link to the 2008 page, and since the latter contains more specific naming guidelines, I think that we're safe keeping the older link.